docs/inferences/ocr.md

```diff
@@ -0,0 +1,46 @@
+A common task for computer vision is extracting text from images, also known as OCR (Optical Character Recognition).
+
+LandingLens has OCR models available out-of-the-box, without the need to train your own model. The models are pre-trained on a variety of fonts types, and are optimized for accuracy and speed.
```

Comment:
Change the beginning to "The LandingLens Python SDK has OCR models".
Reason: The OCR models are only available via the SDK, and not via the LandingLens UI.